--- a/helium.desktop
+++ b/helium.desktop
@@ -3,11 +3,11 @@ Version=1.0
 Name=Helium
 GenericName=Web Browser
 Comment=Access the Internet
-Exec=chromium %U
+Exec=helium %U
 StartupNotify=true
 StartupWMClass=helium
 Terminal=false
-Icon=helium
+Icon=/usr/lib/helium/product_logo_256.png
 Type=Application
 Categories=Network;WebBrowser;
 MimeType=application/pdf;application/rdf+xml;application/rss+xml;application/xhtml+xml;application/xhtml_xml;application/xml;image/gif;image/jpeg;image/png;image/webp;text/html;text/xml;x-scheme-handler/http;x-scheme-handler/https;
@@ -15,8 +15,8 @@ Actions=new-window;new-private-window;
 
 [Desktop Action new-window]
 Name=New Window
-Exec=chromium
+Exec=helium
 
 [Desktop Action new-private-window]
 Name=New Incognito Window
-Exec=chromium --incognito
+Exec=helium --incognito
